src/java_bytecode/java_bytecode_convert_method.h

```diff
@@ -6,6 +6,9 @@ Author: Daniel Kroening, kroening@kroening.com
 
 \*******************************************************************/
 
+/// \file
```

Comment:
This file appears to be missing [this commit](https://github.com/diffblue/cbmc/commit/924e01bd5693f561584ddf67b3ddc77838d698ac), is this intentional/desired @romainbrenguier